### PR TITLE
Add other methods to sample class MyClass

### PR DESCRIPTION
**Pull request by kataXV**
description..

### CODE REVIEW

File: my_class.rb [hunk 1]
```diff
@@ -4,4 +4,16 @@ class MyClass
   def sum(a, b)
     a + b
   end
+
+  def subtract(a, b)
```
Comment (karinamokleva) [inline my_class.rb:8]: what for

File: my_class.rb [hunk 2]
```diff
@@ -4,4 +4,16 @@ class MyClass
...
+    a - b
+  end
+
+  def multiply(a, b)
```
Comment (karinamokleva) [inline my_class.rb:12]: comment 2

File: my_class.rb [hunk 3]
```diff
@@ -4,4 +4,16 @@ class MyClass
...
+    a * b
+  end
+
+  def divide(a, b)
```
Comment (karinamokleva) [inline my_class.rb]:
```suggestion
def div(a, b)
```

**Review by karinamokleva — CHANGES_REQUESTED** [review]

File: my_class.rb (hunk 3 above)
Comment (kataXV) [reply my_class.rb]: no

**Comment by kataXV** [comment]
test

**Pull request merged**